src/EventStore.Core/Services/PersistentSubscription/PersistentSubscriptionService.cs

```diff
@@ -101,7 +101,7 @@ public class PersistentSubscriptionService<TStreamId> :
 		
 		public void Handle(SubscriptionMessage.PersistentSubscriptionsRestart message) {
 			if (!_started) {
-				message.ReplyEnvelope.ReplyWith(new SubscriptionMessage.InvalidPersistentSubscriptionsRestart());
+				message.ReplyEnvelope.ReplyWith(new SubscriptionMessage.InvalidPersistentSubscriptionsRestart("Persistent Subscriptions cannot be restarted as it is in the wrong state."));
```

Comment:
```suggestion
				message.ReplyEnvelope.ReplyWith(new SubscriptionMessage.InvalidPersistentSubscriptionsRestart(
					"The Persistent Subscriptions subsystem cannot be restarted because it is not started."));
```